File: C++/bilangan-terbesar.cpp
#include <iostream>

using namespace std;

int main()
{
    int bil1, bil2, bil3;

    cout << "Inputkan bilangan 1: ";
    cin >> bil1;
    cout << "Inputkan bilangan 2: ";
    cin >> bil2;
    cout << "Inputkan bilangan 3: ";
    cin >> bil3;

    if (bil1 >= bil2 && bil1 >= bil3)
    {
        cout << "Bilangan terbesar adalah " << bil1 << endl;
    }
    else if (bil2 >= bil1 && bil2 >= bil3)
    {
        cout << "Bilangan terbesar adalah " << bil2 << endl;
    }
    else
    {
        cout << "Bilangan terbesar adalah " << bil3 << endl;
    }
}

File: JavaScript/bilangan-terbesar.js
const prompt = require("prompt-sync")({ sigint: true });

let bil1, bil2, bil3;

bil1 = parseInt(prompt("Masukkan bilangan 1: "));
bil2 = parseInt(prompt("Masukkan bilangan 2: "));
bil3 = parseInt(prompt("Masukkan bilangan 3: "));

if (bil1 >= bil2 && bil1 >= bil2) {
	console.log(`Bilangan terbesar adalah ${bil1}`);
} else if (bil2 >= bil1 && bil2 >= bil3) {
	console.log(`Bilangan terbesar adalah ${bil12}`);
} else {
	console.log(`Bilangan terbesar adalah ${bil3}`);
}
